### ISSUE TITLE
Browserify compatibility

### ISSUE THREAD

**Issue by paulpflug** [wtfribley/angular-treemendous#2]
``` js
treemendous = require("angular-treemendous")
angular.module("app", [treemendous])
```

would be wonderful ;)

just add a index.js
with

``` js
require('./angular-treemendous');
module.exports = 'treemendous';
```

and set the main in package.json to index.js

I can make a pr if you want

**Comment by wtfribley** [comment]
This component is already compatible with browserify:

``` js
require('angular-treemendous');
angular.module('app', ['treemendous']);
```

There's no need to expose the name of the component as a string.

What I can fix is the 'main' property of `bower.json` - I must have never updated it when I changed the name of the component. That should solve this issue.

**Comment by wtfribley** [comment]
@paulpflug can you verify that the latest version (0.2.4) works for you as recommended above? If so, I'll close this issue.

**Comment by paulpflug** [comment]
yeah works, the difference between the two is, the first one will be minified. I think it is the recommended version from browserify. but maybe thats not too important ;)

**Comment by wtfribley** [comment]
I'm not sure how this difference would affect minification, but I have been taking a look around at other angular packages and it seems like this method is in wide use... it just feels weird to me to export a string, rather than the module itself. Something like:

``` js
module.exports = angular.module('treemendous', ['ngAnimate'])
```

``` js
var myApp = angular.module('myApp', [
  require('angular-treemendous').name
]);
```

But I'm still debating this... gonna do some research and then implement _something_ better than what I have now. I also need to update dependencies and such - this project requires some general upkeep.

**Comment by paulpflug** [comment]
I read about it at https://github.com/angular/angular.js/pull/10732

For me it is very convincing ;)

**Comment by wtfribley** [comment]
Oh, awesome - this is exactly the discussion I was looking for!

Will update accordingly.
